src/defer-load.directive.ts

```diff
@@ -120,7 +120,7 @@ export class DeferLoadDirective implements OnInit, AfterViewInit, OnDestroy {
 
     private isVisible () {
         let scrollPosition = this.getScrollPosition();
-        let elementOffset = this._element.nativeElement.offsetTop;
+        let elementOffset = this._element.nativeElement.getBoundingClientRect().top + (window.scrollY || window.pageYOffset);;
```

Comment:
could you please remove the double semicolon at the end ?
```suggestion
        let elementOffset = this._element.nativeElement.getBoundingClientRect().top + (window.scrollY || window.pageYOffset);
```